repo-agent/k8s/repowatch-deployment.yaml

```diff
@@ -62,6 +62,19 @@ spec:
       containers:
       - name: repowatch-controller
         image: ko://repo-agent/repowatch/cmd/repowatch-controller # placeholder value, replaced by deployment scripts
```

Comment:
It's good practice to also specify a `securityContext` at the pod level to enforce non-root execution. This enhances the security posture of the pod.

We cannot do read-only root filesystem for container since envbuilder modifies it.